EncodingStreamingHighAvailability/ARMDeployment/azuredeploy.parameters.json

```diff
@@ -0,0 +1,11 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentParameters.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "deploymentNameBase": { "value": "sipetrikha2" },
+    "deploymentRegionLocation": { "value": "eastus" },
+    "keyVaultOwnerObjectId": { "value": "61d47788-7040-4f46-8fa2-59047ac82bb3" },
```

Comment:
I removed keyvaultOwnerObjectId at all. We still need tenant id. We should probably put empty one for actual example. Let's keep this open for now, I will do this change at the end, otherwise I will have to put correct value before each deployment.